### DISCUSSION TITLE
Where can I fill the API key for Weather API

### DISCUSSION THREAD

**Discussion by evdhoek** [mytourbook/mytourbook#1345] [General]
Hello,
I installed my tourbook on a new computer and replaced the folders with the backup.
I tried to import a ride but the weather data was not collected.
@ weather API I asked for a new API but I do not see the screen where it can be filled in and tested.

I tried to add weather manually but get an exeption:
Tour cannot be saved in the database
could not load an entity: [net.tourbook.data.TourData#202451414865156]

Thanks in advance for answering.

**Comment by evdhoek** [comment]
A new import results in the exeption:
 
ClosedByInterruptException
java.nio.channels.ClosedByInterruptException

**Comment by bern6** [comment]
You need to install version 24.5.0

**Reply by evdhoek** [reply]
Thanks very much. Solved!
